lib/portage/package/ebuild/config.py

```diff
@@ -3100,7 +3111,7 @@ def get_virts_p(self):
         self.getvirtuals()
         return self._virtuals_manager.get_virts_p()
 
-    def getvirtuals(self):
+    def getvirtuals(self) -> dict[str, Any]:
```

Comment:
These Anys are, again, hacking around incomplete annotations elsewhere in code this PR partially touches...